blant.c

```diff
@@ -91,14 +91,26 @@ static int _orbitCanonMapping[MAX_ORBITS]; // Maps orbits to canonical (includin
 #define WINDOW_SAMPLE_LEAST_FREQ_MAX 6 // Find the k-graphlet with least fequent cacnonicalInt. IF there is a tie, pick the maximial one 
 int _windowSampleMethod = -1;
 
-#define WINDOW_COMBO 0 // Turn on for using Combination method to sample k-graphlets in Window. Default is DFS-like way.
+#define WINDOW_ITER_COMB 1 // using Combination method to sample k-graphlets in Window
+#define WINDOW_ITER_DFS 2 // Default way: using DFS-like way.
+int _windowIterationMethod = WINDOW_ITER_DFS;
+
+#define WINDOW_LIMIT_UNDEF 0
+#define WINDOW_LIMIT_DEGREE 1
+#define WINDOW_LIMIT_EDGES 2
```

Comment:
It would be good if we could define more than one property--which requires these values to be powers of 2 in order to "OR" them together bitwise. This is fine for now but be sure, if new ones are added, to use powers of two, and add a comment to that effect.